### ISSUE TITLE
New Unblockers

### ISSUE THREAD

**Issue by VFXKLIPS** [bypassiwastaken/skiovox-helper#135]
I need a new unblocker , or a securly disabler for 2024 because my friends chromebook updated and they want one.

**Comment by DankFireR** [comment]
i dont think this is a good github page to be looking for such things, perhaps try going to [ext-remover](https://github.com/3kh0/ext-remover/discussions)?

**Comment by JackWagon885** [comment]
version, kernver, & board please
